### PR TITLE
Reactive spines reacts to burns

### PR DESCRIPTION
**Pull request by carshalash**
For a while there have been some blobs that are a bit lacklaster compared to others, this is the second of my series of blob chemical updates/buffs to make them more suited to handle the modern space mans weaponry. Considering that my more SLIP focused pressurized slime update seemed to be pretty well received I hope that these changes will continue to be liked.

Reactive spines is a little bit of a joke, the only real brute melee weaponry that will be used against it is a captains saber, and if that's the case the crew is probably screaming at him for being close to the blob to begin with. My proposed change is to give reactive spines the ability to also react to burn damage, limiting people using welders and eswords against it and try to properly make reactive spines a more ranged only blob.

Tl;dr
welderbad

:cl:
balance: Causes the reactive spine blob type to retaliate to bun as well as brute damage
/:cl:

### CODE REVIEW

**Comment by Fabian272** [comment]
how are you supposed to fight it then, maxcap it from a distance?

**Comment by carshalash** [comment]
Maybe use guns.

**Comment by necromanceranne** [comment]
Reactive Spines is one of the 'Don't go near it' blob. Most are like this, but this one is explicitly telling you it gets free hits when harmed by brute, so you never, ever want to approach it on the off chance some idiot smacks it with a toolbox, or you are using shotguns to chip away at it.

If you were entering melee against Reactive Spines before, you honestly were proving just how much of blob operates on Darwinist theory. Even if burn damage bypasses the passive effect, it genuinely isn't worth the risk.

**Comment by necromanceranne** [comment]
Also, update the description for the strain.

**Comment by lordpidey** [comment]
> Causes the reactive spine blob type to retaliate to bun as well as brute damage

Are people assaulting the blob with buns?  They deserve death for their wastage of bread products!
